CHANGES.rst

```diff
@@ -2,6 +2,13 @@
 Changelog
 =========
 
+
+Unreleased
+==========
+
+- Add documentation about TinyMCE styles and formats [davilima6]
+
+
```

Comment:
I think documentation repository is one exception to the rule of always adding a change log entry :-)